commons/src/main/scala/dagr/commons/io/Io.scala

```diff
@@ -42,17 +42,20 @@ trait IoUtil {
   val StdOut  = PathUtil.pathTo("/dev/stdout")
   val DevNull = PathUtil.pathTo("/dev/null")
 
+  /** How large a buffer should be used when buffering operations. */
+  def bufferSize: Int = 32 * 1024
```

Comment:
It can't easily be a long - all the underlying APIs take ints, presumably because they are using a byte[] or similar to do buffering, and arrays are limited to length == Int.MAX_VALUE.